sketch_oct9a.ino

```diff
@@ -256,7 +294,12 @@ void setup() {
       //The string returned from readStringUntil does not include the terminator
       String command = Serial1.readStringUntil('\n');
 
-      if (command == "wake") {
+      if (command == "start") {
```

Comment:
Didn't know where else to put this comment. I understand the logic in 'setup()' that it will never exit this 'do-while' loop, so that's effectively our 'loop' function. I don't understand how this will compile because we don't have a dedicated 'loop' function though. Could you explain that to me?

Reply:
We do have a dedicated loop function (`void loop()` on line 326, as well as two other loops that run concurrently with it). After receiving the start command, the `break;` statement causes it to exit the while loop in setup.

Reply:
ahh, gotcha. I didn't see that because that wasn't changed and that was the one section that I didn't manually pull down to look at it.